cluster/common.sh

```diff
@@ -950,6 +950,11 @@ EOF
     if [ -n "${TENANT_SERVER_KUBECONFIGS:-}" ]; then
       cat >>$file <<EOF
 TENANT_SERVER_KUBECONFIGS: $(yaml-quote ${TENANT_SERVER_KUBECONFIGS})
+EOF
+    fi
+    if [ -n "${RESOURCE_SERVER_KUBECONFIG:-}" ]; then
+      cat >>$file <<EOF
+RESOURCE_SERVER_KUBECONFIG: $(yaml-quote ${RESOURCE_SERVER_KUBECONFIG})
```

Comment:
only one RP config ? or multiple ones in the variable ?

Reply:
Only one, we don't need this parameter once we got the RP controller csr issue fix.

Reply:
Removed in new commit